lib/core-server/src/core-presets.test.ts

```diff
@@ -203,4 +203,21 @@ describe('dev cli flags', () => {
       );
     });
   });
+
+  describe('Invalid staticDir must throw: root directory /', () => {
+    const optionsWithInvalidStaticDir = {
+      ...cliOptions,
+      staticDir: ['/'],
+    };
+    beforeEach(() => {
```

Comment:
do we need these? it's already happening in the outer describe. also can remove from the above tests